src/openapi/streaming/subscription.js

```diff
@@ -205,9 +208,16 @@ function onSubscribeSuccess(referenceId, result) {
     }
     this.updatesBeforeSubscribed = null;
 
+    this.retryCounter = 0;
+
     onReadyToPerformNextAction.call(this);
 }
 
+function retrySubscription() {
+    this.onUnsubscribe();

```

Comment:
It could be that a proxy timed out, and the subscription was actually successful. I _assume_ the server copes with receiving an unsubscribe request for a non-existent subscription if that's not indeed the case.